### PR TITLE
CALCOM-8009 - [CAL-1371] /settings - team members: Show people's booking link

### LINKED ISSUE #8009: [CAL-1371] /settings - team members: Show people's booking link
Currently it shows the email and no username

edit by @PeerRich: show "•" instead of "|" as separator

We should also show the booking link of all team members.

Designs:

[View in Figma →](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A69603&t=GyqQV5A60KRJ3jdf-1)

**Make sure to use our** [**list item component **](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=27844%3A90422&t=GbbAFEFKW3r9VpPC-1)**@sean-brydon built.**

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1371](https://linear.app/calcom/issue/CAL-1371/settings-team-members-show-peoples-booking-link)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
Fixes #8009 

image

### CODE REVIEW

**Comment by sean-brydon** [comment]
Can we use the vertical divider like the figma? 

[View in Figma →](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A69603&t=GyqQV5A60KRJ3jdf-1)

**Comment by gitstart-calcom** [comment]
> Can we use the vertical divider like the figma?
> 
> [View in Figma →](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=26%3A69603&t=GyqQV5A60KRJ3jdf-1)

Hi @sean-brydon, we made the changes based on @PeerRich's comment [here](https://github.com/calcom/cal.com/issues/8009#issuecomment-1490228018)
Screenshot 2023-03-31 at 11 08 43 AM

**Comment by PeerRich** [comment]
lets hide if === null
image

File: packages/features/ee/teams/components/MemberListItem.tsx [hunk 1]
```diff
@@ -112,12 +114,16 @@ export default function MemberListItem(props: Props) {
...
-              </span>
+              <div className="flex items-center">
+                <span
+                  className="block text-sm text-gray-600"
+                  data-testid="member-email"
+                  data-email={props.member.email}>
+                  {props.member.email}
+                </span>
+                <span className="mx-2 block text-gray-600">•</span>
+                <span className="block text-sm text-gray-600">{bookingLink}</span>
```
Comment (PeerRich) [inline packages/features/ee/teams/components/MemberListItem.tsx:125]: lets hide these two if === null

Comment (PeerRich) [inline packages/features/ee/teams/components/MemberListItem.tsx:125]:
```suggestion
                {bookingLink && <>
                  <span className="mx-2 block text-gray-600">•</span>
                  <span className="block text-sm text-gray-600">{bookingLink}</span>
                </>
                }
```

Comment (PeerRich) [reply packages/features/ee/teams/components/MemberListItem.tsx:125]: this probably needs to re-run linter after commit

**Review by PeerRich — APPROVED** [review]

**Comment by zomars** [comment]
This PR seems to have introduced a regression

image

**Pull request merged**